hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1475,6 +1475,10 @@ public String getGraphiteMetricPrefix() {
     return getString(HoodieMetricsGraphiteConfig.GRAPHITE_METRIC_PREFIX_VALUE);
   }
 
+  public int getGraphiteMerticReportPeriodSeconds() {
```

Comment:
too long....
What about renaming to `getGraphiteReportPeriodSeconds `?

Reply:
Ah, it's okay